ruma-federation-api/src/query.rs

```diff
@@ -1,4 +1,5 @@
 //! Endpoints to retrieve information from a homeserver about a resource.
 
+pub mod custom;
```

Comment:
This doesn't seem too different from the other endpoints here and should probably have a similar name. Maybe `get_custom_information`?

Reply:
Sounds good